Userland/Libraries/LibCrypto/BigFraction/BigFraction.h

```diff
@@ -6,6 +6,7 @@
 
 #pragma once
 
+#include <AK/Error.h>
 #include <AK/DeprecatedString.h>
```

Comment:
Also, I think it would be nice to make the LibCrypto commit message more specific since, from a simple grep, I do see that LibCrypto/Cipher/AES.h has its own `to_deprecated_string()`. You can just change the commit message to have the category of "LibCrypto/BigFraction: ...."